src/pages/contact-us.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react'
+
+import Layout from '../components/Layout'
+import ConstrainedWidth from '../components/Layout/ConstrainedWidth'
+
+import Text from '../components/Text'
+
+import VerticalSpacing from '../components/VerticalSpacing'
+import PageMeta from '../components/PageMeta'
+
+import Container from '../components/Container'
+import Twitter from '../components/twitter.svg'
+import PageTop from '../components/PageTop'
+import contactArcs from '../components/contactArcs.svg'
+import EmailLink from '../components/EmailLink'
+const ContactUs = () => (
+  <Layout>
+    <PageMeta title="Contact us" description="Enter in suitable description" />
+    <PageTop>
+      <ConstrainedWidth>
+        <Text size="xlarge" color="#48e9ce">
+          <h1>Get in touch</h1>
+        </Text>
+
+        <div className="contact-header">
+          <Text size="normal">
+            <div className="subtitle">
+              Drop us an email. We’d love to have a conversation about how we
+              could work with you.
+            </div>
+          </Text>
+          <div className="main-arc-container">
+            <img src={contactArcs} height={136} width={191} />
+          </div>
+        </div>
+
+        <EmailLink
+          link="/"
+          background="#5600ee"
+          border_color="#5600ee"
+          mobileSize="0.9rem"
+        >
+          Email: hello@neontribe.co.uk
+        </EmailLink>
+      </ConstrainedWidth>
+    </PageTop>
+    <VerticalSpacing size={5} />
+
+    <ConstrainedWidth>
+      <Text size="normal" color="black" maxCharacter="49ch">
+        Follow us on Twitter
+      </Text>
+
+      <div className="social-media-container">
+        <div className="social-media-labels">
+          <img src={Twitter} height={42} width={42}></img>
+
+          <Text color="#561dee" size="normal">
+            <h4>@neontribe</h4>
+          </Text>
+        </div>
+      </div>
+
+      <Text size="normal" color="black" maxCharacter="55ch">
+        We share our clients work and best practice in the tech for good sector.
+      </Text>
+    </ConstrainedWidth>
+
+    <VerticalSpacing size={3} />
+    <div className="white-background">
+      <ConstrainedWidth paddingTop="3em">
+        <Container mobileFlexDirection="column">
+          <div>
+            <Text color="black" size="normal" maxCharacter="53ch">
+              We work from home as well as in the office. Please check before
+              you plan a visit.
+            </Text>
+            <VerticalSpacing size={3}></VerticalSpacing>
+            <Container justifyContent="flex-start">
+              <Text color="#561dee">
```

Comment:
Because I am a nerd and love using all the html elements, here would be a nice place to use the `<address>` element, which is specifically for the purpose of indicating the address details of an person or organisation. Totally optional fun added extra, you can checkout the docs here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address